packages/utils/src/surescripts/shared.ts

```diff
@@ -29,14 +29,20 @@ export function getSurescriptsDirOrFail(): string {
   return dir;
 }
 
-export function writeSurescriptsRunsFile(filePath: string, content: string): void {
+export function writeSurescriptsRunsFile(filePath: string, content: string): string {
   const dir = getSurescriptsDirOrFail();
   const fullFilePath = path.join(dir, filePath);
   const fileDir = path.dirname(fullFilePath);
   if (!fs.existsSync(fileDir)) {
     fs.mkdirSync(fileDir, { recursive: true });
   }
   fs.writeFileSync(fullFilePath, content, "utf-8");
+  return fullFilePath;
+}
+
+export function appendToSurescriptsRunsFile(fullFilePath: string, content: string): string {
+  fs.appendFileSync(fullFilePath, content, "utf-8");
+  return fullFilePath;
 }
```

Comment:
nit; imo functions should either do io or return a value, not both. I'd just pass in the full filePath to these functions instead of creating it inside and then returning it.

Reply:
Addressed

Reply:
i know you said addressed but i still see the file path returned from the writeSurescriptsRunsFile. I was mainly talking about that one - create and pass in the path from the caller